oeps/oep-0045-arch-ops-and-config.rst

```diff
@@ -0,0 +1,175 @@
+##########################################
+OEP-45: Configuring and Operating Open edX 
+##########################################
+
+.. list-table::
+
+   * - OEP
+     - :doc:`OEP-0045 </oeps/oep-0045>`
+   * - Title
+     - Configuring and Operating Open edX
+   * - Last Modified
+     - 2020-04-06
+   * - Authors
+     - Bill DeRusha
+   * - Arbiter
+     - Felipe Montoya 
+   * - Status
+     - Under Review
+   * - Type
+     - Architecture
+   * - Created
+     - 2020-04-06
+   * - Review Period
+     - 2020-04-17 - 2020-05-04
+
+.. contents::
+   :local:
+   :depth: 3
+
+
+Abstract
+========
+
+Wherever possible, The edX organization will provide and manage Docker images as the medium for packaging operating dependencies of Open edX applications rather than providing Ansible playbooks.
+
+The configuration for running applications in different environments will be done through a config file per unique [application, environment] combination. How those files are generated and managed is the responsibility of each operator.
```

Comment:
For completeness, can you define "application"?  Is it equivalent to an independently deployable service?  Does it include `Django App Plugins` or feature libraries?  (I'm assuming you're referring to IDAs here.)

Reply:
Would Open edX platform services be clearer?

Reply:
changed to IDAs